charts/spire/charts/spiffe-oidc-discovery-provider/templates/deployment.yaml

```diff
@@ -47,7 +53,7 @@ spec:
             - containerPort: 8008
               name: healthz
           {{- if not .Values.insecureScheme.enabled }}
-            - containerPort: 443
+            - containerPort: 8443
```

Comment:
port switched to an unprivileged port so it can run as nonroot. The service can stay 443 so only the pod needs to change.